user.js

```diff
@@ -1389,7 +1389,7 @@ user_pref("mail.chat.notification_info", 2);
  * [SETTING] Calendar > Calendar > Timezone ***/
 user_pref("calendar.timezone.local", "UTC");  // [DEFAULT: ""]
 /* 9313: Disable calendar service performing event "extraction" from email content ***/
-user_pref("calendar.extract.service.enabled", false);  // [DEFAULT: false]
+// user_pref("calendar.extract.service.enabled", false);  // [DEFAULT: false]
```

Comment:
The rationale is here is the same for `mail.collect_email_address_outgoing` : I vote for keeping features that copy incoming/outgoing data _elsewhere_, where most of users forget about them and not control their storage.

Reply:
I am not sure, does it copy the events somewhere else? If this is the case then for sure keep it in this template.

But I thought its just the service that ***prompts*** users to add it to their calendar, opt-in

Reply:
> I am not sure, does it copy the events somewhere else? If this is the case then for sure keep it in this template.

I guess events stay in the calendar even once you have deleted the e-mail where they come from ? If so, I assume they are copied in calendar own database. (full disclosure : I never used this feature so I may be mistaken here)

> But I thought its just the service that _**prompts**_ users to add it to their calendar, opt-in

Thanks for the detail, taken into account :+1:

Reply:
So I've caught a glimpse of the code related to event extraction and it's hundreds of lines. I've also noticed the setting is actually disabled be default, so let's reset this pref :+1: